src/test/java/uk/gov/hmcts/reform/profilesync/repository/ProfileSyncAuditRepositoryTest.java

```diff
@@ -2,18 +2,18 @@
 
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.extension.ExtendWith;
```

Comment:
Is there a reason we are introducing outdated junit 4 constructs to replace current junit 5 test setup? (lines 5-16)